test/e2e/testsuites/pre_provisioned_provided_credentials_tester.go

```diff
@@ -41,32 +41,43 @@ type PreProvisionedProvidedCredentiasTest struct {
 func (t *PreProvisionedProvidedCredentiasTest) Run(client clientset.Interface, namespace *v1.Namespace) {
 	for _, pod := range t.Pods {
 		for n, volume := range pod.Volumes {
-			accountName, accountKey, accountSasToken, containerName, err := t.Driver.GetStorageAccountAndContainer(context.Background(), volume.VolumeID, nil, nil)
+			accountName, accountKey, _, containerName, err := t.Driver.GetStorageAccountAndContainer(context.Background(), volume.VolumeID, nil, nil)
 			framework.ExpectNoError(err, fmt.Sprintf("Error GetStorageAccountAndContainer from volumeID(%s): %v", volume.VolumeID, err))
+			pod.Volumes[n].ContainerName = containerName
+			var secreteData map[string]string
+
+			var run = func() {
```

Comment:
`run()` only provisions different secret, I think we need to add a new e2e test case for sas token stored in k8s secret scanerio

Reply:
`run()` will generate different secret, pvc, pv and pod and also create and clean them up in k8s each time. Also per the name of this test case: `pre_provisioned_provided_credentials_tester`,I think this test case is used to test the scenario of provided credentials, which includes access key, sas token, service principal and managed identity. I think it would be more reasonable and convenient to put them in this test case.